errorHandler.js

```diff
@@ -0,0 +1,53 @@
+const { HttpCodes } = require("./messages/httpCodes");
+const { customErrors } = require("./messages/errors");
+const { messages } = require("./messages/messages");
+
+class MessageToUser {
```

Comment:
Looks like there is better to separate successful message logic an error message logic to different files.

- [ ] create file like`userMessage.js` and put class `MessageToUser` to that file